docs/teams/mentoring.md

```diff
@@ -1,68 +1,174 @@
----
-original:
-  authors: Fiona Gillan
-  url: https://github.com/fgillan/sr-team-coord/wiki/Mentoring
----
 # Mentoring
```

Comment:
A lot of this new content is super useful, although it feels slightly duplicated of https://studentrobotics.org/runbook/volunteering/mentor-guidance/. Perhaps worth combining the pages and cross-referencing?

Reply:
My goal with this page is to capture the "what" of mentoring, as a guide for organising it as a whole and for people who haven't done it before (including people who haven't heard of it before). I think the focus of the other one is much more "how".
That said, I haven't re-reviewed the other one in detail to compare the overlap. I suspect that moving the other one into this teams folder could be a useful thing to do, but deliberately didn't change that here.